### ISSUE TITLE
Left Most Focus Issue

### ISSUE THREAD

**Issue by rahat14** [egeniq/android-tv-program-guide#62]
I have a two view one is left column then the EPG Program 
![Screenshot 2024-01-12 025518](https://github.com/egeniq/android-tv-program-guide/assets/27925309/ab1e43f0-e963-4aba-a0c3-deb06a680f3a)
Marked Area written in compose but the EPG part done on the XML. Now The problem is that it is not passing the focus from epg to left coulm but the foucs from right to left is working.
@dzolnai  please help 
![Screenshot 2024-01-12 025518](https://github.com/egeniq/android-tv-program-guide/assets/27925309/b618e2e0-8cd8-46ab-975d-90cbd818a597)

**Comment by dzolnai** [comment]
Are the left column and the EPG in the same fragment, or do you use nested fragments?

**Comment by rahat14** [comment]
yes, they are in the same compose, The EPG container is wrapped in Andrroid View and the other is in lazy list and both of them are in Row. 
I also tried in XML configuration, where the left column stays in a linear layout, the epg fragment in linear layout  under same parent

**Comment by dzolnai** [comment]
Can you send me a reproduction project where I can see this happening?

**Comment by rahat14** [comment]
sure  I can send  the  whole code
